tables/subject_plasma/singh22-plasma.csv

```diff
@@ -0,0 +1,9 @@
+﻿ref_name,subject_name,rx_name,collection_date_cmp,collection_date,location,cumulative_group,section
+Singh22,CP_ChAdOx_1dose,CP_ChAdOx_1dose_3wk,~,2021-04-15,India,CP_ChAdOx_1dose,NULL
+Singh22,ChAdOx_1dose,ChAdOx_1dose_3wk,~,2021-04-15,India,ChAdOx_1dose,NULL
+Singh22,CP,CP_pre__infection,~,2021-03-15,India,CP,NULL
+Singh22,CP,CP_post_infection,~,2021-04-15,India,CP,NULL
+Singh22,ChAdOx_1dose_seronegative,ChAdOx_1dose_seronegative_before_vaccination,~,2021-03-15,India,ChAdOx_1dose_seronegative,NULL

```

Comment:
@zachandfox, if the plasma sample was extracted before infection or before vaccination, please ignore the results.